### ISSUE TITLE
Can't find NoiseAsset_256X256_PNG.png 

### ISSUE THREAD

**Issue by asklar** [microsoft/CsWinRT#693]
1. Created a C# NET 5 class library. Reference the WinUI 3 Preview 4 CI 1/26 nuget package. 
2. Follow the authoring instructions to produce a nuget pkg for the class library
3. Created a cppwinrt WinUI 3 app from WinUI 3 Preview 3 VSIX 
4. update cppwinrt app's WinUI reference to latest internal WinUI nuget preview 4 from 01/26
5. Referenced the NET 5 class library nupkg file from the cppwinrt app via a local nuget source
6. Build solution

## Bug 1:
> 1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4858,5): error MSB3030: Could not copy the file "C:\Users\asklar\source\repos\NET5Lib\packages\NET5Lib.1.0.0\lib\native\Microsoft.UI.Xaml\Assets\NoiseAsset_256X256_PNG.png" because it was not found.

Workaround for now: create an empty png file at that location

**Comment by huoyaoyuan** [comment]
The issue should be cpp or package side? None of these files are used by a C# project.

**Comment by j0shuams** [comment]
I believe some of these are related to this line: https://github.com/microsoft/CsWinRT/blob/e33691ea5f4bac9baaaaa3be969ebab4772cc90d/nuget/Microsoft.Windows.CsWinRT.targets#L156

My idea is to make a new item group that is essentially filter on ReferenceCopyLocalPaths for just WinRT.Runtime and Microsoft.Windows.SDK.NET (which are the only two we want to include from a cswinrt perspective)

**Comment by asklar** [comment]
thanks @j0shuams - what's the best way to track these? should I split these into separate bugs for more granular tracking?
I've updated the original bug description as I've been able to make a bit more progress with an updated VSIX. I'm currently blocked on being able to activate any type in the C# library from C++, we get a WinRT fail fast when loading the dll.

**Comment by j0shuams** [comment]
Thanks for breaking them up @asklar I prefer having an individual issue for each bug, makes it easier to track progress. 

I am thinking about what could be the cause of your blocker. Two immediate thoughts come to me: are you using WinUI in your app or just plain C#? If using WinUI, then you may need to update to a newer winui version... If not using WinUI, I would be interested in seeing what's in your c++ app's output dir. If it wants to use a C# library via winmd (made by cswinrt) then the following dlls must be in the outdir (where the c++ app's exe gets placed) -- 
- WinRT.Host.dll
- WinRT.Host.Shim.dll
- WinRT.Runtime.dll
- YourC#Library.dll
- Microsoft.Windows.SDK.NET.dll
- YourC++App.exe.manifest
- WinRT.runtimeconfig.json

**Comment by asklar** [comment]
@j0shuams Thanks. The app is using the latest internal WinUI 3 VSIX (will be out as part of Preview 4 when it is made public). The png file that it mentions is part of the WinUI nuget package, but something about how the files in the package are copied back and forth is broken for this particular file.
So to clarify how things are set up:
- NET5Lib class library in C#, references WinUI 3 Preview 4 internal nuget, uses CSWinRT, produces NET5Lib.nupkg
- App1 cppwinrt app, references NET5Lib nuget package and WinUI 3 Preview 4 internal nuget package.

**Comment by angelazhangmsft** [comment]
Seem to be seeing this issue again with consuming by package reference in a C# .NET 5 WinUI App

Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3030	Could not copy the file "C:\Users\angzhang\.nuget\packages\winuiauthoringsample\1.0.1\lib\net5.0-windows10.0.18362\Microsoft.UI.Xaml\Assets\NoiseAsset_256X256_PNG.png" because it was not found.	CsWinUIDesktop	C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets	4919

**Comment by soroshsabz** [comment]
I have the same issue

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not copy the file "C:\Users\ssoroosh\.nuget\packages\microsoft.ui.xaml\2.8.0\lib\uap10.0\Microsoft.UI.Xaml\Assets\NoiseAsset_256X256_PNG.png" because it was not found.	Linphone			
```

**Comment by huoyaoyuan** [comment]
Microsoft.UI.Xaml 2.x doesn't use CsWinRT. It should be indeed another issue.
